pkg/kube/util/resources/fetch.go

```diff
@@ -0,0 +1,21 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	crc "sigs.k8s.io/controller-runtime/pkg/client"
+
+	qsv1a1 "code.cloudfoundry.org/quarks-secret/pkg/kube/apis/quarkssecret/v1alpha1"
+)
+
+// ListQuarksSecrets fetches all the quarkssecrets from the namespace
+func ListQuarksSecrets(ctx context.Context, client crc.Client, namespace string) (*qsv1a1.QuarksSecretList, error) {
```

Comment:
The only advantage of this method, is that it's wrapping the error. Probably duplicating the error message..

Reply:
Removed whole GetReconciles package.